plugins/prune.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: prune
+spec:
+  version: v0.1.0
+  platforms:
+  - selector:
+      matchExpressions:
+      - {key: os, operator: In, values: [darwin, linux]}
+    uri: https://github.com/thecloudnatives/kubectl-plugins/archive/v0.1.0.zip
+    sha256: 563d17508b37f56d668c2942922568cbaa014557651e69aedfcb8afda5a5f32c
+    files:
+    - from: kubectl-plugins-*/prune/kubectl-prune.sh
+      to: kubectl-prune
```

Comment:
Somehow, `krew` creates a folder here instead of renaming the file. @ahmetb is that a known issue / desired behavior?

You can work around as follows:
```yaml
files:
- from: kubectl-plugins-*/prune/kubectl-prune.sh
  to: "."
bin: ./kubectl-prune.sh
```